### PR TITLE
TPROD-322 - PageBundle - Convert PHP Templates to Twig Templates

### PR DESCRIPTION
**Pull request by JoshuaEstes**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 

- Does searching for a page appear in the global search?
- /s/pages work? Sort, buttons, etc.
- Does the results page work? filter, sort
- [results] export as csv, excel, html?
- Details page load correctly
- [details] has A/B tests
- [details] has translations
- [details] A/B test stats modal work correctly
- Preview page loads correctly
- Add A/B test to existing landing page
- Create new landing page
- Edit existing landing page
- Does the builder function correctly?
- Within Builder, do all slot types work correctly?
- Within builder, do all section types work correctly?

### CODE REVIEW

**Review by LordRembo — COMMENTED** [review]
I can't really comment on the PHP code changes but I tested the provided scenarios (make a page, search for a page, …) and they all work as expected.
One thing I wasn't able to test, is downloading results, as I couldn't generate any. Not an issue with this PR, as that's not related to templates. Just my local installation misbehaving it seems.

**Comment by mabumusa1** [comment]
- [X]  Pages Dashboard 
- [X]  Create a Category 
- [X] Create a Page with published_at, unpublished_at, scripts
- [X] Page Dashboard
- [X] Builder loads without issues
- [X] Page loads correctly and the page preview
- [X] Edit page 
- [X] Graph in the dashboard shows the numbers
- [X] A/B Test works
- [X] A/B Stats modal work
- [X] Translation works
- [X] Clone works 
- [X] Publish/Unpublish Ajax works 
- [X] Page submission dashboard works
- [X] Download Sumbission works  
- [X] Global Search works
- [X] Builder works with all slots
- [X] Builder works with tokens

**Review by mabumusa1 — APPROVED** [review]
The PR works as expected, thanks @JoshuaEstes

**Comment by mabumusa1** [comment]
https://github.com/all-contributors please add @LordRembo  for testing

**Comment by mabumusa1** [comment]
@allcontributors please add @LordRembo for testing

**Comment by mabumusa1** [comment]
@allcontributors please add @LordRembo for userTesting

**Comment by mabumusa1** [comment]
@allcontributors please add @LordRembo for userTesting

File: app/bundles/PageBundle/Views/FormTheme/Point/pointaction_urlhit_widget.html.twig [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    'i' => $view['translator']->trans('mautic.core.time.minutes'),
+    'H' => $view['translator']->trans('mautic.core.time.hours'),
+    'd' => $view['translator']->trans('mautic.core.time.days'),
+];
+
+?>
+
+<div class="row">
+    <div class="col-xs-12">
+        <?php echo $view['form']->row($form['page_url']); ?>
```
Comment (escopecz) [inline app/bundles/PageBundle/Views/FormTheme/Point/pointaction_urlhit_widget.html.twig:21]: @JoshuaEstes What's the plan with this template? It has PHP syntax within the Twig file.

Comment (JoshuaEstes) [reply app/bundles/PageBundle/Views/FormTheme/Point/pointaction_urlhit_widget.html.twig:21]: No idea how I missed this

https://github.com/mautic/mautic/pull/12031

**Pull request merged**